scripts/delegate-command.mjs

```diff
@@ -0,0 +1,33 @@
+/**
+ * Delegate the provided command to a sub-project.
+ * For example:
+ * `npm run in vue-handsontable build` or `run in vue build`
+ * will run the `build` command from the directory of the provided project.
+ */
+import execa from 'execa';
+
+const [project, command, ...args] = process.argv.slice(2);
+const PROJECT_ALIASES = {
+  angular: 'angular-handsontable',
+  react: 'react-handsontable',
+  vue: 'vue-handsontable'
+};
+const commandArray = [
+  'run',
+  command
+];
+
+if (args) {
+  commandArray.push(...args);
+}
+
+((async function() {
+  try {
+    await execa('npm', commandArray, {
+      cwd: (project === 'handsontable' ? '.' : `./wrappers/${PROJECT_ALIASES[project] || project}`),
```

Comment:
It's worth adding a notice that for other projects - those that will be placed in a different directory than `wrappers` and root the script won't work. 

The script could already handle projects from other directories. As well this can be polished after this project phase.